apps/expo/src/components/tracker/__snapshots__/set-entry-header.test.tsx.snap

```diff
@@ -0,0 +1,49 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SetEntryHeader should render correctly 1`] = `
+<View
+  className="mb-2 flex flex-row items-center justify-between px-2"
```

Comment:
Could you put all the attributes and closing brackets on the same line as the elements? Inlining the attributes is fine as long as the line length doesn't get too long.

Reply:
this is a snapshot for testing, its being generated so its good